metricbeat/module/kubernetes/controllermanager/_meta/fields.yml

```diff
@@ -0,0 +1,124 @@
+      description: >
+        Infrastructure zone
+    - name: process
+      type: group
+      fields:
+        - name: cpu.sec
+          type: double
+          description: CPU seconds
+        - name: memory.resident.bytes
+          type: long
+          description: Bytes in resident memory
+        - name: memory.virtual.bytes
+          type: long
+          description: Bytes in virtual memory
+        - name: fds.open.count
+          type: long
+          description: Number of open file descriptors
+        - name: started.sec
+          type: double
+          description: Seconds since the process started
+    - name: http
+      type: group
+      fields:
+        - name: request.duration.us.percentile.*
+          type: object
+          object_type: double
+          description: Request duration microseconds percentiles
+        - name: request.duration.us.sum
+          type: double
+          description: Request duration microseconds cumulative sum
+        - name: request.duration.us.count
+          type: long
+          description: Request count for duration
+        - name: request.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Request size percentiles
+        - name: request.size.bytes.sum
+          type: long
+          description: Request size cumulative sum
+        - name: request.size.bytes.count
+          type: long
+          description: Request count for size
+        - name: response.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Response size percentiles
+        - name: response.size.bytes.sum
+          type: long
+          description: Response size cumulative sum
+        - name: response.size.bytes.count
+          type: long
+          description: Response count
+        - name: request.count
+          type: long
+          description: Request count for response
+    - name: client.request.count
+      type: long
+      description: >
+        Number of requests as client
+    - name: workqueue
+      type: group
+      fields:
+        - name: longestrunning.sec
+          type: double
+          description: Longest running processors
+        - name: unfinished.sec
+          type: double
+          description: Unfinished processors
+        - name: adds.count
+          type: long
+        - name: depth.count
+          type: long
+        - name: retries.count
+          type: long
+          description: Unfinished processors
+    - name: node.collector
+      type: group
+      fields:
+        - name: eviction.count
+          type: long
+          description: Number of node evictions
+        - name: unhealthy.count
+          type: long
+          description: Number of unhealthy nodes
+        - name: count
+          type: long
+          description: Number of nodes
+        - name: health.pct
+          type: long
+          description: Percentage of healthy nodes
```

Comment:
Can these percentage have a decimal part? Then it shouldn't be long.
It'd be good if we could make it in the scale from 0 to 1 so we can use `format: percent`.

Reply:
not that I know